queenbee/schema/workflow.py

```diff
@@ -30,8 +31,14 @@ class Workflow(BaseModel):
 
     id: str = str(uuid4())
 
+    info: Info = Field(
```

Comment:
re: calling this item `metadata`

Reply:
We can! I didn't want to create a conflict if the key is used by K8. Otherwise I think `metadata` is the better choice.

Reply:
None of this information should make it's way into k8s right?